gctrpc/rpc.swagger.json

```diff
@@ -1218,12 +1009,6 @@
             "schema": {
               "$ref": "#/definitions/gctrpcOrderDetails"
             }
-          },
-          "default": {
-            "description": "An unexpected error response",
```

Comment:
Are you using the latest version of protobufs? Not sure why these are being removed